fdbserver/MutationTracking.cpp

```diff
@@ -103,12 +102,13 @@ TraceEvent debugTagsAndMessageEnabled(const char* context, Version version, Stri
 			BinaryReader br(mutationData, AssumeVersion(rdr.protocolVersion()));
 			OTELSpanContextMessage scm;
 			br >> scm;
-		} else if (EncryptedMutationMessage::startsEncryptedMutationMessage(mutationType)) {
-			throw encrypt_unsupported();
 		} else {
 			MutationRef m;
 			BinaryReader br(mutationData, AssumeVersion(rdr.protocolVersion()));
 			br >> m;
+			if (m.isEncrypted()) {
+				throw encrypt_unsupported();
```

Comment:
This just returns a trace event for debugging so it probably should not throw.  

It might be best to just change nothing here.  `m` will be passed to `debugMutation()` which will ignore the `Encrypted` type.

Reply:
I want to make sure an encrypted mutation is not pass into MutationTracking by accident. I think changing it to an assert would be better. What do you think?